sdks/python/apache_beam/transforms/window.py

```diff
@@ -376,6 +376,10 @@ def windowed_value(
     # type: (...) -> WindowedValue
     return WindowedValue(value, timestamp, (GlobalWindow(), ), pane_info)
 
+  @classmethod
+  def windowed_value_at_end_of_window(cls, value):
+    return cls.windowed_value(value, GlobalWindow().max_timestamp())
```

Comment:
should this be `end` instead of `max_timestamp`?

Reply:
`end` is the exclusive end. We do want this to lie in the window in question, so max_timestamp is correct.